test/services/messengerSync/utils.test.js

```diff
@@ -118,4 +118,32 @@ describe('messenger synchronizer utils', () => {
 			expect(result.rooms.length).to.eq(3);
 		});
 	});
+
+	describe('messengerActivatedForSchool', () => {
+		it('true if messenger flag is set', async () => {
+			// arrange
+			this.app = app;
+			const school = await testObjects.createTestSchool({ features: ['messenger'] });
+			const user = await testObjects.createTestUser({ roles: ['teacher'], schoolId: school._id });
+
+			// act
+			const result = await messengerActivatedForSchool({ userId: user._id});
+
+			// assert
+			expect(result).to.equal(true);
+		});
+
+		it('false if messenger flag is not set', async () => {
+			// arrange
+			this.app = app;
+			const school = await testObjects.createTestSchool({ });
+			const user = await testObjects.createTestUser({ roles: ['teacher'], schoolId: school._id });
+
+			// act
+			const result = await messengerActivatedForSchool({ userId: user._id});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required before '}'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5362179)